capstone/src/main/java/com/google/sps/servlets/announcements/Announcement.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps.servlets;
```

Comment:
In general, object and corresponding classes should remain in the main servlets directory. This is because these objects will likely be used by more than one servlet so it's good to keep them in shared/common directories. If you want, you can make a "common" or "shared" directory to put all of these shared classes in (eg: objects, constants, etc.)